.github/workflows/setup/action.yml

```diff
@@ -35,6 +35,10 @@ runs:
         run: cargo install -f rustup-toolchain-install-master hyperfine
         shell: bash
 
+      - name: Install nightly toolchain
+        run: rustup toolchain install nightly --profile minimal
+        shell: bash
```

Comment:
Why do we now need this?

Reply:
I suppose more than anything I'm confused that we didn't need this before.

Reply:
We now need this because miri-script needs nightly because it picks up the `.config/cargo.toml` which uses nightly features.

Previously we didn't need this because miri-script was built with stable, and then it used rustup-toolchain-install-master to install the toolchain used for Miri.

Reply:
Ah.